includes/admin/post-transition.php

```diff
@@ -93,6 +101,13 @@ function publish_tweet( $post_id ) {
 		return;
 	}
 
+	/**
```

Comment:
```suggestion
	/*
```

Docblocks for files, functions and filers require the double asterisks, multiline comments only require a single asterisk.

Reply:
Thanks for the note on this.